repo/packages/M/myriad/0/resource.json

```diff
@@ -0,0 +1,9 @@
+{
+  "images" : {
+    "icon-small" : "https://raw.githubusercontent.com/apache/incubator-myriad/master/website/img/myriad_elephant.png",
+    "icon-medium" : "https://raw.githubusercontent.com/apache/incubator-myriad/master/website/img/myriad_elephant.png",
+    "icon-large" : "https://raw.githubusercontent.com/apache/incubator-myriad/master/website/img/myriad_elephant.png"
```

Comment:
please use the image specifications listed here so they show up properly in dc/os
https://github.com/mesosphere/universe#images